homeassistant/config.py

```diff
@@ -244,18 +246,20 @@ def set_time_zone(time_zone_str):
 
     set_customize(config.get(CONF_CUSTOMIZE) or {})
 
-    if CONF_TEMPERATURE_UNIT in config:
-        hac.temperature_unit = config[CONF_TEMPERATURE_UNIT]
+    if CONF_UNIT_SYSTEM in config:
+        hac.units = IMPERIAL_SYSTEM \
```

Comment:
I don't think this style really helps with readability much. 

```python
if config[CONF_UNIT_SYSTEM] == SYSTEM_IMPERIAL:
    hac.units = IMPERIAL_SYSTEM
else:
    hac.units = METRIC_SYSTEM
```

And actually, by writing this out, we need better naming if we are having `SYSTEM_IMPERIAL` and `IMPERIAL_SYSTEM`. That's confusing.